src/Accounts/Accounts/help/Get-AzContext.md

```diff
@@ -26,7 +26,7 @@ Get-AzContext [-ListAvailable] [-RefreshContextFromTokenCache] [-DefaultProfile
 ## DESCRIPTION
 The Get-AzContext cmdlet gets the current metadata used to authenticate Azure Resource Manager requests.
 This cmdlet gets the Active Directory account, Active Directory tenant, Azure subscription, and the targeted Azure environment.
-Azure Resource Manager cmdlets use these settings by default when making Azure Resource Manager requests.
+Azure Resource Manager cmdlets use these settings by default when making Azure Resource Manager requests. Please notes that the contexts in list are populated when current account logins by `Connect-AzAccount`.
```

Comment:
If a subscription does not show up in the results of `Get-AzContext -ListAvailable`, it may be because the number of contexts has exceeded the default limit of 25. Please run `Connect-AzAccount -MaxContextPopulation <int>` and specify a larger limit.